README.md

```diff
@@ -70,3 +71,13 @@ Once installed, tests can be run from the root directory of the workflow. These
 ``` sh
 pytest
 ```
+
+## References
+
+This pipeline uses code and infrastructure developed and maintained by the [nf-core](https://nf-co.re) initiative, and reused here under the [MIT license](https://github.com/nf-core/tools/blob/master/LICENSE).
+
+> The nf-core framework for community-curated bioinformatics pipelines.
+>
+> Philip Ewels, Alexander Peltzer, Sven Fillinger, Harshil Patel, Johannes Alneberg, Andreas Wilm, Maxime Ulysse Garcia, Paolo Di Tommaso & Sven Nahnsen.
+>
+> Nat Biotechnol. 2020 Feb 13. doi: 10.1038/s41587-020-0439-x.
```

Comment:
Great addition! I was thinking we needed to add references as well.

Reply:
Absolutely!